test/fixtures/output/kafka/link/list-link-yaml.golden

```diff
@@ -26,3 +26,10 @@
   state: ""
   error: ""
   error_message: ""
+- link_name: link-5
+  source_cluster_id: ""
+  destination_cluster_id: ""
+  remote_cluster_id: cluster-2
+  state: ""
+  error: ""
+  error_message: ""
```

Comment:
`omitempty`?

Reply:
Pre-existing. I don't think we should change it at this point if consumers assume the empty String.